From 1f107f0dddb8617e025c3241899b9edcfcaa3a49 Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Tue, 27 Aug 2013 19:25:43 +0200
Subject: [PATCH 5/5] linux: fix initialization of tstamp_precision_list

We should be using tstamp_precision_list to store time stamp precisions
which we support.
---
 pcap-linux.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/pcap-linux.c b/pcap-linux.c
index ee1ef05..5856dc4 100644
--- a/pcap-linux.c
+++ b/pcap-linux.c
@@ -425,13 +425,13 @@ pcap_create_interface(const char *device, char *ebuf)
 	 * adapters?
 	 */
 	handle->tstamp_precision_count = 2;
-	handle->tstamp_type_list = malloc(2 * sizeof(u_int));
-	if (handle->tstamp_type_list == NULL) {
+	handle->tstamp_precision_list = malloc(2 * sizeof(u_int));
+	if (handle->tstamp_precision_list == NULL) {
 		free(handle);
 		return NULL;
 	}
-	handle->tstamp_type_list[0] = PCAP_TSTAMP_PRECISION_MICRO;
-	handle->tstamp_type_list[1] = PCAP_TSTAMP_PRECISION_NANO;
+	handle->tstamp_precision_list[0] = PCAP_TSTAMP_PRECISION_MICRO;
+	handle->tstamp_precision_list[1] = PCAP_TSTAMP_PRECISION_NANO;
 #endif /* defined(SIOCGSTAMPNS) && defined(SO_TIMESTAMPNS) */
 
 	return handle;
-- 
1.8.3.1

